File: 1006/Solution.java
class Solution {
    public int clumsy(int n) {
        int res = n--;
        if(n > 0) res *= n--;
        if(n > 0) res /= n--;
        if(n > 0) res += n--;
        while(n > 0){
            int cur = n--;
            if(n > 0) cur *= n--;
            if(n > 0) cur /= n--;
            res -= cur;
            if(n > 0) res += n--;
        }
        return res;
    }
}

File: 1006/Solution.go
func clumsy(n int) int {
    res := n
    n--
    if n > 0 {
        res *= n
        n--
    }
    if n > 0 { 
        res /= n
        n--
    }
    if n > 0 {
        res += n
        n--
    }
    for n > 0 {
        
        cur := n
        n--
        if n > 0 {
            cur *= n
            n--
        }
        if n > 0 {
            cur /= n
            n--
        }
        res -= cur
        if n > 0 {
            res += n
            n--
        }
    }
    return res
}
